odfdom/src/test/java/org/odftoolkit/odfdom/doc/table/TableTest.java

```diff
@@ -885,7 +885,8 @@ public void testGetCellByPosition() throws Exception {
     table = mOdsDoc.getTableByName("Sheet1");
     cell = table.getCellByPosition("C1");
     Assert.assertNotNull(cell);
-    Assert.assertEquals("Currency", cell.getStringValue());
+    // FixMe: Testfile now with pretty printing in XML https://github.com/tdf/odftoolkit/issues/229
```

Comment:
I don't understand this comment. Does the pretty-printing change the string value?